tests/gnrc_udp/udp.c

```diff
@@ -87,7 +87,6 @@ static void send(char *addr_str, char *port_str, char *data_len_str, unsigned in
     uint16_t port;
     ipv6_addr_t addr;
     size_t data_len;
-
```

Comment:
Please leave this whitespace as is.

Reply:
Thank you for your time and feedback @jcarrano ! I have made the necessary changes in the file, and update my commits as well!

Reply:
I believe that I haven't used the static test tools to verify the quality of changes which resulted in whitespaces modifications. The change I made on top of it didn't resolve it, so I will make a fresh commit again after taking care of it ! Thanks